docs/contributing.md

```diff
@@ -64,6 +64,8 @@ Patches should be submitted as [pull requests](https://help.github.com/articles/
 Before submitting a PR:
 - Your code must run and pass all the automated tests before you submit your PR for review. "Work in progress" pull requests are allowed to be submitted, but should be clearly labeled as such and should not be merged until all tests pass and the code has been reviewed.
   - Run `cargo test --all` to make sure all tests still pass and no warnings are emitted.
+  - Run `cargo run -p sync-test` to make sure the integration sync test still pass.
+  You need to have [node](https://nodejs.org/en/) installed for running the integration test.
```

Comment:
for l10n the url can be just https://nodejs.org

Reply:
Done